components/MapView.js

```diff
@@ -399,6 +399,10 @@ class MapView extends React.Component {
     this._runCommand('fitToSuppliedMarkers', [markers, animated]);
   }
 
+  fitToCoordinates(coordinates, edgePadding, animated) {
```

Comment:
Maybe we could also add some documentation to this method that explains the possible keys of `edgePadding`.  Either with a comment, or again using default arguments as documentation:

```js
fitToCoordinates(coordinates, edgePadding = { top = 0, right = 0, bottom = 0, left = 0 }, animated = true) {
```